src/mpyl/project.py

```diff
@@ -534,9 +534,19 @@ def job(self) -> Job:
     def project_yaml_path() -> str:
         return "deployment/project.yml"
 
+    @staticmethod
+    def project_overrides_yml_pattern() -> str:
+        return "deployment/project-override-*.yml"
+
     @property
     def root_path(self) -> str:
-        return self.path.replace(Project.project_yaml_path(), "")
+        if self.path.endswith(".yml"):
+            try:
+                return str(Path(self.path).parents[1])
+            except IndexError:
+                return self.path
+        else:
+            return self.path
```

Comment:
```suggestion
        if self.path.endswith(".yml"):
            try:
                return str(Path(self.path).parents[1])
            except IndexError:
                pass
        return self.path
```

A bit more clean maybe?